ldms/man/ldmsctl.man

```diff
@@ -469,6 +469,16 @@ The updater name. If none is given, the statuses of all updaters are
 reported.
 .RE
 
+.SS Query the updater match list
+.BR updtr_match_list
+attr=<value>
+.RS
+.TP
+.BI [name " name"]
+.br
+The updater name. If none is given, the match list of all updaters are reported.
```

Comment:
Should we update this to say the "list of sets matching the updtr match regular expression" is reported. Is this correct?

Reply:
No. I'm reporting the regular expressions. However, it is more precise to say that the command returns the list of regular expressions to match set names or set schemas.